camayoc/tests/qpc/api/v1/authentication/test_login.py

```diff
@@ -27,18 +27,35 @@ def test_login():
         to build our authentication headers and make authenticated requests.
     """
     client = api.Client(authenticate=False)
-    client.login()
+    response = client.login()
+    data = response.json()
+    assert data.get("token"), "No authentication 'token' received"
     client.get(QPC_SOURCE_PATH)
 
 
+def test_logout():
+    """Test that we can logout to the server.
```

Comment:
Something minor, but I believe "Test that we can log out of the server". logout is a noun and log out is a verb.